File: z-clients/javascript/src/models/svixPollerCreateOut.ts
// this file is @generated

export interface SvixPollerCreateOut {
    topic: string;
    pollerId: string;
}

export const SvixPollerCreateOutSerializer = {
    // biome-ignore lint/suspicious/noExplicitAny: intentional any
    _fromJsonObject(object: any): SvixPollerCreateOut {
        return {
            topic: object['topic'],
            pollerId: object['poller_id'],
        };
    },

    // biome-ignore lint/suspicious/noExplicitAny: intentional any
    _toJsonObject(self: SvixPollerCreateOut): any {
        return {
            'topic': self.topic,
            'poller_id': self.pollerId,
        };
    }
}

File: z-clients/rust/src/models/svix_poller_create_out.rs
// this file is @generated
use serde::{Deserialize, Serialize};

#[derive(Clone, Debug, Deserialize, Serialize)]
pub struct SvixPollerCreateOut {
    pub topic: String,

    pub poller_id: String,
}

impl SvixPollerCreateOut {
    pub fn new(topic: String, poller_id: String) -> Self {
        Self { topic, poller_id }
    }
}
